tests/test_pit.py

```diff
@@ -153,8 +166,8 @@ def test_expr(self):
                    2019-07-15               0.000000      0.000000               0.047369              0.094737                               0.047369
                    2019-07-16               0.000000      0.000000               0.047369              0.094737                               0.047369
                    2019-07-17               0.000000      0.000000               0.047369              0.094737                               0.047369
-                   2019-07-18               0.175322      0.175322               0.135029              0.094737                               0.135029
```

Comment:
Could you please explain why this should be `0.087661` instead of `0.135029`? 
According to the content of data [here](https://qlib.readthedocs.io/en/latest/advanced/PIT.html#file-based-design-for-pit-data).

The mean of last two quarters (201901, 201902)  on 2019-07-18  should be the average of   `(0.094737 + 0.175322) /2 = 0.1350295`.
Which is not the same as the changed version you committed.
Thanks

![image](https://user-images.githubusercontent.com/465606/187165013-4dc2d3a8-ac8a-444a-9159-5d59fc3be40d.png)

Reply:
I used the last two data to calculate the average:
`(1.75322 + 0) / 2 = 0.087661`

Reply:
`Mean($$roewa_q, 2)` operator does not declare an observation point, why use 1.75322 instead of 0?

Reply:
> `Mean($$roewa_q, 2)` means calculate the average of the **latest value** of the last two quarters at the current observation time point. The last two quarters are `201901` and `201902`. Its latest value is `0.094737` and `0.175322` respectively.
> 
> @Chaoyingz

I don't think the `Mean` operator has so much meaning, it just calculates the last two values of the given df (it doesn't matter the quarter).

Reply:
@Chaoyingz   
Yes, it just calculates the last two values of a given df.
But each row in the df indicates the value in a specific quarter in the  PIT quarter data.

There are a lot of typical use cases.  For example,  the average XXX of the last 4 quarters.

Reply:
Having a lot of typical use cases does not mean that we need to change the purpose of the `Mean` operator. I think it is possible to write an operator specifically to handle this use case. And it should be difficult to implement this function in the current `PitStorage`.